service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/NotificationService.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.hmcts.reform.fpl.service.email;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.service.notify.NotificationClient;
+import uk.gov.service.notify.NotificationClientException;
+
+import java.util.Map;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class NotificationService {
+    private final NotificationClient notificationClient;
+
+    public void sendNotification(String templateId, String email, Map<String, Object> parameters, String reference) {
```

Comment:
this name is quite generic, from parameters we know that we are sending email, maybe sendEmail is more precise name